src/_helpers/lib_info_helper.js

```diff
@@ -165,9 +165,9 @@ export function getLibraryCodeFromUrl(lib_url, debug=false, defLibCode="help"){
   return lib_code;
 }
 
-export const getAllLibraryInfo = function(){ return libraries_data_backup; }
+export const getAllLibraryInfo = () => { return libraries_data_backup; }
```

Comment:
arrow functions are 100% ES6 syntax. you'll probably have to remove `const` here.